desktop/apps/categories2/components/Sidebar.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react'
 
-export default class Sidebar extends React.Component {
-  render(){
-    const { data } = this.props
```

Comment:
Yea, with this we do not even need to store data from props into the variable.

Just commenting here so that I remember the suggestion I've heard about this syntax. This would allow us to assign multiple const from props in case needed. Also the suggestion was to keep the local name to be the same as props name so that it's easier to navigate the code.